core/Pos/Core/Genesis/Canonical.hs

```diff
@@ -386,7 +386,7 @@ instance ReportSchemaErrors m => FromJSON m (ProxySecretKey EpochIndex) where
         pskIssuerPk <- fromJSField obj "issuerPk"
         pskDelegatePk <- fromJSField obj "delegatePk"
         pskCert <- fromJSField obj "cert"
-        return ProxySecretKey {..}
+        pure UnsafeProxySecretKey{..}
```

Comment:
Why are you sure that this psk is valid?

Reply:
I'm not sure it's valid, but stuff like `VssCertificate` isn't checked in `Pos.Core.Genesis.Canonical` either so I assumed that whoever wrote `Pos.Core.Genesis.Canonical` knew what they were doing. Maybe it was a wrong assumption in hindsight. If you want, I can change it.